files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,113 @@
+#5= IFCPOSTALADDRESS(.OFFICE.,'Office postal address',$,$,('StreetAddress 1','StreetAddress 2'),'Box 1','Town','Region','Box 1','Country');
+#6= IFCORGANIZATION('TRMB','Trimble Inc.','Trimble consulting',(#4),(#5));
+#7= IFCPERSONANDORGANIZATION(#3,#6,$);
+#8= IFCAPPLICATION(#6,'version','Application full name','Application Identifier');
+#9= IFCOWNERHISTORY(#7,#8,$,.NOCHANGE.,$,$,$,1681466043);
+#10= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#11= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#12= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#13= IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#14= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#15= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#16= IFCSIUNIT(*,.SOLIDANGLEUNIT.,$,.STERADIAN.);
+#17= IFCSIUNIT(*,.POWERUNIT.,$,.WATT.);
+#18= IFCUNITASSIGNMENT((#10,#11,#12,#13,#14,#15,#16,#17));
+#19= IFCCARTESIANPOINT((-200.,200.,0.));
+#20= IFCDIRECTION((0.,0.,1.));
+#21= IFCDIRECTION((1.,0.,0.));
+#22= IFCAXIS2PLACEMENT3D(#19,#20,#21);
+#23= IFCDIRECTION((0.725975,0.687721));
+#24= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,1.E-06,#22,#23);
+#25= IFCPROPERTYSINGLEVALUE('Project data',$,IFCLABEL('Project data 1.'),$);
+#26= IFCPROPERTYSINGLEVALUE('Length property',$,IFCLENGTHMEASURE(1.),#10);
+#27= IFCPROPERTYSET('2VvPInIaH4DwJhCV6wOt4Z',#9,'Project pset',$,(#25,#26));
+#28= IFCARCHELEMENTTYPE('15kDej$Jj5WRaa0jByCU3H',#9,'Name',$,$,$,$,$,$,.NOTDEFINED.);
+#29= IFCGROUP('1U1QQhW69A_8bBUGHUmG8K',#9,'TypeObject Group',$,$);
+#30= IFCRELASSIGNSTOGROUP('1rf9RG57H9jxvJeMb$RIi_',#9,$,$,(#28),$,#29);
+#31= IFCPROPERTYENUMERATION('DrillingDiameter',(IFCPOSITIVELENGTHMEASURE(100.)),$);
+#32= IFCSIUNIT(*,.LENGTHUNIT.,.MILLI.,.METRE.);
+#33= IFCSIMPLEPROPERTYTEMPLATE('3WdblmOunBkObDtVCYCj33',#9,'DrillingDiameter',$,.P_SINGLEVALUE.,'IfcPositiveLengthMeasure',$,#31,#32,$,$,$);
+#34= IFCPROPERTYSETTEMPLATE('13qDND5F9BCwZ4atfD3ipQ',#9,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBoreHole',(#33));
+#35= IFCPERSON($,$,'Given name',$,$,$,$,$);
+#36= IFCACTOR('3niGQH_or1FOnohcRd9J0b',#9,'Actor',$,$,#35);
+#37= IFCPROJECT('3KF6hmUH9ExfmDglggWKPu',#9,'Project name','\X2\987976EE7B804ECB\X0\',$,$,$,(#24),#18);
+#38= IFCRELDECLARES('1$0XJheM99GeDbfDYRG6_4',#9,'TypeObjects',$,#37,(#28));
+#39= IFCRELDECLARES('13Ff5NhErDWhECcbQfoaDo',#9,'PropertysetTemplates',$,#37,(#34));
+#40= IFCRELDECLARES('3gCndEcEz6pRL6ZSS3CAd_',#9,'Actors',$,#37,(#36));
+#41= IFCRELDECLARES('0sNoOyBcj60efR3VSigS79',#9,'IfcGroup',$,#37,(#29));
+#42= IFCCARTESIANPOINT((0.,0.,0.));
+#43= IFCAXIS2PLACEMENT3D(#42,#20,#21);
+#44= IFCLOCALPLACEMENT($,#43);
+#45= IFCSITE('0jf8GwS$L3YP5fxsJ17CHX',#9,'Site name',$,$,#44,$,$,.ELEMENT.,(60,11,10),(24,48,23),21590.,$,$);
+#46= IFCCLASSIFICATION('Source',$,$,'Name',$,$,$);
+#47= IFCRELASSOCIATESCLASSIFICATION('3oXo6BC7j7iOeSRjyj0mp$',#9,$,$,(#37),#46);
+#48= IFCPROJECTEDCRS('EPSG:3879','ETRS89 / GK25FIN','EPSG:3879',$,$,$,$);
+#49= IFCMAPCONVERSION(#24,#48,5000.,1520.,6500.,1.1,2.1,1.,2.,3.);

```

Comment:
A scale factor of `3.` would mean that the map meters are three times as long as model meters. While technically this is OK, I wonder if this is intended?

Reply:
Just curious :) I will change back to 1.